### PR TITLE
HTML4 Converted To HTML5.

### PR DESCRIPTION
**Pull request by mmertgunduz**
_Hello, I converted the system to HTML5 :-)_

The things I will add:

🟢 Handbook PDF (31/03/2022)
🟢  The whole website will be changed from HTML4 to HTML5. (15/04/2022)
🔴 Syntatic errors will be fixed. (Not Done)
🔴 CSS changes. (Not Done)
🔴 JS will be added. (Not Done)
🔴 Mobile Responsivity   (Not Done)
🔴 SEO operations (Not Done)

----------------------------------
🟢 --> Completed
🔴 --> Not Completed

### CODE REVIEW

File: handbook/handbook.in [hunk 1]
```diff
@@ -143,7 +143,7 @@ Depending on the size, creating the file system can take several minutes.</p>
 
 <h2>Overview</h2>
 
-<p><img src="images/nomadbsd-overview.png" alt="" /></p>
+<p><img src="handbook/images/nomadbsd-overview.png" alt="" /></p>
```
Comment (mrclksr) [inline handbook/handbook.in]: The image source is not correct. Either use the relative path, `images/nomadbsd-overwiew.png`, or use an absolute path, `@ROOTDIR@handbook/images/nomadbsd-overview.png`.

File: handbook/handbook.in [hunk 2]
```diff
@@ -287,7 +287,7 @@ programs are automatically executed when the graphical interface starts.
 Further, it allows you to enable/disable some components of the NomadBSD
 desktop. The changes take place after logging out and in again.</p>
 
-<p><img src="images/dsbautostart-screenshot.png" alt="" />
+<p><img src="handbook/images/dsbautostart-screenshot.png" alt="" />
```
Comment (mrclksr) [inline handbook/handbook.in]: Same here.

File: handbook/handbook.in [hunk 3]
```diff
@@ -309,7 +309,7 @@ the theme's directory to <em>/usr/local/share/slim/themes/</em>. To see a previe
 in <em>nomadbsd-dmconfig</em> copy a screenshot of the login screen to
 <em>/usr/local/share/slim/themes/your-theme-name/preview.png</em>.</p>
 
-<p><img src="images/nomadbsd-dmconfig-screenshot.png" alt="" />
+<p><img src="handbook/images/nomadbsd-dmconfig-screenshot.png" alt="" />
```
Comment (mrclksr) [inline handbook/handbook.in]: Same here.

File: handbook/handbook.in [hunk 4]
```diff
@@ -438,7 +438,7 @@ the position, resolution, etc. of your monitors. Save your changes to
 allows you to change the brightness, gamma, screen mode, display power management
 (DPMS) settings, etc.</p>
 
-<p><img src="images/dsbdisplaysettings-screenshot.png" alt="" /></p>
+<p><img src="handbook/images/dsbdisplaysettings-screenshot.png" alt="" /></p>
```
Comment (mrclksr) [inline handbook/handbook.in]: Same here.

File: handbook/handbook.in [hunk 5]
```diff
@@ -521,7 +521,7 @@ single partition is currently not possible.</p>
 
 <hr />
 
-<p><img src="images/installer-screenshot.png" alt="" /></p>
+<p><img src="handbook/handbook/images/installer-screenshot.png" alt="" /></p>
```
Comment (mrclksr) [inline handbook/handbook.in]: Same here.

File: template [hunk 6]
```diff
@@ -21,29 +16,16 @@
 	<div id="navbar">
 		<ul>
 			<li id="navHome"><a href="@ROOTDIR@index.html">Home</a></li>
-		</ul>
-		<ul>
-			<li id="navHandbook"><a href="@ROOTDIR@handbook/handbook.html">Handbook</a></li>
-		</ul>
-		<ul>
+			<li id="navHandbook"><a href="@ROOTDIR@handbook.html">Handbook</a></li>
```
Comment (mrclksr) [inline template]: The href path is not correct. Must be `@ROOTDIR@handbook/handbook.html`

File: template [hunk 7]
```diff
@@ -1,18 +1,13 @@
...
 <head>
-	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-	<meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate"/>
-	<meta http-equiv="Pragma" content="no-cache"/>
-	<meta http-equiv="Expires" content="0"/>
-	<link rel="stylesheet" href="@ROOTDIR@style.css" content="text/css" media="screen"/>
+	<meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css"/>
```
Comment (mrclksr) [inline template]: href must be `@ROOTDIR@style.css`

**Comment by mrclksr** [comment]
Thank you for the pull request, Mehmet :-)

I've added a few notes to the problematic lines. Except for the path problems, it is merge ready.

File: handbook/handbook.in (hunk 1 above)
Comment (mmertgunduz) [reply handbook/handbook.in]: Thanks for the great feedback! Fixed :-)

File: handbook/handbook.in (hunk 2 above)
Comment (mmertgunduz) [reply handbook/handbook.in]: Thanks for the great feedback! Fixed :-)

File: handbook/handbook.in (hunk 3 above)
Comment (mmertgunduz) [reply handbook/handbook.in]: Thanks for the great feedback! Fixed :-)

File: handbook/handbook.in (hunk 5 above)
Comment (mmertgunduz) [reply handbook/handbook.in]: Thanks for the great feedback! Fixed :-)

File: template (hunk 7 above)
Comment (mmertgunduz) [reply template]: Thanks for the great feedback! Fixed :-)

**Comment by mmertgunduz** [comment]
Thanks for your great feedback Marcel! I love working with you, always fast and very detailed feedbacks :-)

Fixed the bugs and commited!

**Review by mrclksr — APPROVED** [review]

**Comment by mrclksr** [comment]
Thank you, Mehmet :-)

**Pull request merged**